.evergreen/config.yml

```diff
@@ -424,6 +424,14 @@ buildvariants:
       ssl: "ssl"
     then:
       remove_tasks: ".3.6"
+  # workaround for CDRIVER-3318 preventing us from doing any version checks
```

Comment:
both `buildInfo` and `serverStatus` seem to hang indefinitely on SSL sharded clusters on MacOS so we have no way to do a version check. this makes running a lot of the tests impossible so I've just removed these variants for now.